.github/workflows/fedora-interop.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM fedora:38
+RUN dnf -y update
+RUN dnf -y install perl
+RUN dnf -y install git tmt-all
+RUN dnf -y remove openssl
+RUN cd / && git clone https://gitlab.com/not4pedro/interop
```

Comment:
Please use `https://gitlab.com/redhat-crypto/tests/interop`

Reply:
I don't see the point of having a custom container. `tmt` provides its own on `quay.io/testing-farm/tmt-all`.

Reply:
@quarckster I didn't realize there was one.  If its capable of also doing builds, yeah, I'll switch to that

Reply:
@quarckster the tmt-all container doesn't support gcc at the moment.  It can be added at run time of course, but that sort of makes it a 6 of one / 0.5 of a dozen of the other situation.  Either we modify an existing container, or we build our own, Not sure which is preferable, but I would lean towards building our own container, as it gives us the possibility to fix bugs in the container more easily